independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json

```diff
@@ -0,0 +1,597 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NASA Image and Video Library",
+    "description": "Use this API to access the NASA Image and Video Library site at images.nasa.gov",
+    "version": "1.0",
+	"contact": {

```

Comment:
@paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.